storybook/preview.js

```diff
@@ -63,13 +64,26 @@ export const globalTypes = {
 			],
 		},
 	},
+	wrapper: {
+		name: 'Wrapper',
+		description: 'Wrap the component in a div with a max-width.',
+		defaultValue: 'none',
+		toolbar: {
+			icon: 'outline',
```

Comment:
Open to suggestions on a better icon choice. I'd have used the group one but it's in use for the viewport resizing toolbar item.